Pyjion/pyjit.cpp

```diff
@@ -517,53 +564,136 @@ void PyjionJitFree(PyjionJittedCode* function) {
         delete code;
         g_pyjionJittedCode.erase(function);
     }
-#else
-    delete (TraceInfo*)function->j_evalstate;
 #endif
+	delete obj;
+}
+
+static PyObject *pyjion_enable(PyObject *self, PyObject* args) {
+	auto ts = PyThreadState_Get();
+	auto prev = ts->interp->eval_frame;
+	ts->interp->eval_frame = PyJit_EvalFrame;
+	return prev == PyJit_EvalFrame ? Py_False : Py_True;
+}
+
+static PyObject *pyjion_disable(PyObject *self, PyObject* args) {
+	auto ts = PyThreadState_Get();
+	auto prev = ts->interp->eval_frame;
+	ts->interp->eval_frame = _PyEval_EvalFrameDefault;
```

Comment:
Technically this could be inaccurate if a previous frame evaluation function was set, but I don't know if we want to create module state just for that one case.

If we document this anywhere we should be upfront that we blindly clobber the frame evaluation function.